<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Stream Control</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Stream Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="streamTimer">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sampling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Update Rate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="updateRate">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> Hz</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Chunk Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="chunkSize">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 63)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> points</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Data History Length:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="historyLength">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="suffix">
             <string> points</string>
            </property>
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>5000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_1">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="plotContainer"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
